src/components/Toast/index.js

```diff
@@ -84,29 +96,36 @@ export default class Button extends PureComponent {
     } = this.props
 
     const {
-      show,
       height,
+      show,
     } = this.state
 
     const classNames = cn({
-      'mc-toast mc-mt-2 mc-py-1 mc-px-3': true,
-      'mc-toast--show': show,
+      'mc-toast': true,
+      'mc-toast--appear': show,
```

Comment:
Same thing, different name.  This just keeps inline with the `CSSTransition` naming